boards/arm/samv7/same70-xplained/Kconfig

```diff
@@ -90,4 +90,38 @@ config SAME70XPLAINED_HSMCI0_AUTOMOUNT_UDELAY
 
 endif # SAME70XPLAINED_HSMCI0_AUTOMOUNT
 
+menuconfig SAME70XPLAINED_APP_FORMAT_MCUBOOT
+	bool "MCUboot-bootable format"
+	default n || MCUBOOT_BOOTLOADER
+	depends on BOOT_MCUBOOT
+	---help---
+		The SAMV7 port of MCUboot supports the loading of unsegmented firmware
+		images.
```

Comment:
It seems this help text was based on the one from ESP32. The standard Image Format for ESP32 defines image segments, which differs from the MCUboot image format (that explains the "unsegmented" term).

You may want to rewrite this help text to adequate to this different context.

Reply:
I think that segmented firmware image is not unique to ESP32. In MCUboot terminology it is called "Multiple image boot". I mean that if firmware is segmented it can be loaded using "Multiple image boot", but not that those two terms are equal. I can update text to avoid "unsegmented" term if needed.

Reply:
"Multiple image boot" is unrelated to the segmentation we are discussing here. The **segments** relate to the data and code sections from a given executable firmware image. Think about the sections of an ELF file.
The ESP32 image format organizes these sections (segments) differently, optimized for the chip's memory organization: https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/app_image_format.html
So, although it might not be unique to ESP32, that's the segmentation I was referring to.

The MCUboot's "Multiple image boot" is literally support for multiple firmware images, but **per image slot**. So this enables the device to support the execution of a different application firmware image, e.g. a recovery firmware image as the second image on the slot.

Reply:
Agree. Thank for the clarification. Now I see that "Multiple image boot" does not handle segmented load, but again I think that "segmented" firmware is not ESP32 specific, but more related to code located in multiple memory regions that is usually a use case when code is executed from RAM. I see that MCUboot "RAM loading" support the RAM load for a single segment of code and in case if we need a multiple segment load the additional development should be done on MCUboot side.

Reply:
> [...] , but again I think that "segmented" firmware is not ESP32 specific, but more related to code located in multiple memory regions that is usually a use case when code is executed from RAM.

That's what I tried to explain in my previous answer, but I might not have made myself clear.
**Segments** per se are not exclusive to ESP32. Memory segments are specified in the ELF file format.
But that specific "unsegmented" term on the help text should refer to a characteristic of the Espressif file format, and that probably does not make sense to state in this context because that's no different than the non-MCUboot behavior for this chip.
For a ESP32 "segmented image" the information for each segment is stored in the Image Header, which allows the segments to be juxtaposed in the binary file, resulting in a denser binary file.
The "unsegmented" term should mean that the MCUboot image format for ESP32 does not follow the same approach. Instead it relies on the VMA and LMA information provided via linker file, similar to the linker scripts from other chip vendors.

Reply:
Done